docs/source/api.rst

```diff
@@ -243,6 +243,52 @@ Chalice
         entire lambda function name.  This parameter is optional.  If it is
         not provided, the name of the python function will be used.
 
+   .. method:: on_sqs_message(queue, batch_size=1, name=None)
+
+      Create a lambda function and configure it to be automatically invoked
+      whenever a message is published to the specified SQS queue.
+
+      The lambda function must accept a single parameter which
+      is of type :class:`SQSEvent`.
+
+      If the decorated function returns without raising any exceptions
```

Comment:
Yeah I ran into this as well and meant to document.  I think this makes the most sense in the topic guide rather than the API docs for `on_sqs_message`, thoughts?

Reply:
Yeah that sounds good to me. It is pretty annoying that the default queue you create in the console is incompatible with the default chalice lambda function. It would be nice if we accounted for that with managed resources.

Reply:
Yeah I pushed a change with doc updates.